e2e/cli_test.go

```diff
@@ -5,6 +5,7 @@ package e2e
 import (
 	"bytes"
 	"encoding/json"
+	"github.com/styrainc/regal/internal/util"
```

Comment:
nit: import order

Reply:
Interesting. I wonder why it's not caught by `golangci-lint run ./...` 🤔 But fixed now either way!